src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -144,7 +144,7 @@ private void StartServer()
             // and so that if any messages are logged after Process is set to null we know which process it related to.
             processId = Process.Id;
             logger.WriteLine(Resources.INFO_ServerProcessId, processId);
-            logger.LogDebug($"[eslint-bridge] [process id: {processId}] Server process HasExited: {1}.", processId, Process.HasExited);
```

Comment:
Miscellaneous: I messed up converting from the resource string format to the interpolated string format. The method was logging "HasExited: 1".